src/migrate/Migrator.js

```diff
@@ -295,9 +299,17 @@ export default class Migrator {
   // passing any `variables` given in the config to the template.
   _writeNewMigration(name, tmpl) {
     const { config } = this;
-    const dir = this._absoluteConfigDir();
+    const dirs = this._absoluteConfigDir();
+    if (dirs.length > 1) {
```

Comment:
Maybe here it would be more usable to use the last migration dir where to put new migration and add that to documentation. Otherwise one would always need separate configuration just for being able to create new migration...

Reply:
Good idea!